pages/linux/atool.md

```diff
@@ -1,20 +0,0 @@
-# atool
-
-> Manage archives of various formats.
-> More information: <https://manned.org/atool>.
-
-- List files in a Zip archive:
-
-`atool {{[-l|--list]}} {{path/to/archive.zip}}`
```

Comment:
Add the missibg option placeholders as is seen in the original page

Reply:
Sure! Original page was missing few useful examples so I didn't use it as is.